src/test/scala/nl.knaw.dans.easy.multideposit/CustomMatchers.scala

```diff
@@ -39,14 +39,17 @@ trait CustomMatchers {
   def containTrimmed(content: String) = new ContentMatcher(content)
 
   class EqualTrimmedMatcher(right: Iterable[Node]) extends Matcher[Iterable[Node]] {
+    private val pp = new PrettyPrinter(160, 2)
+
+    private def prepForTest(n: Node) = XML.loadString(pp.format(n))
+
+    private def pretty(ns: Iterable[Node]) = ns.map(n => XML.loadString(pp.format(n))).mkString("\n")
+
     override def apply(left: Iterable[Node]): MatchResult = {
       MatchResult(
-        left.zip(right).forall { case (l, r) =>
-          val pp = new PrettyPrinter(160, 2)
-          XML.loadString(pp.format(l)) == XML.loadString(pp.format(r))
-        },
-        s"$left did not equal $right",
-        s"$left did equal $right"
+        left.zip(right).forall { case (l, r) => prepForTest(l) == prepForTest(r) },
+        s"${ pretty(left) } was not equal to ${ pretty(right) }",
+        s"${ pretty(left) } was equal to ${ pretty(right) }"
```

Comment:
prepForTest prettifies and the string prettifies. Perhaps you can avoid repeating yourself.

Reply:
👍 for spotting that!
I'll fix it in both PRs.